src/puppet-web/firer.ts

```diff
@@ -54,13 +54,13 @@ const regexConfig = {
 
   roomJoinInvite: [
     // There are 3 blank(charCode is 32) here. eg: You invited 管理员 to the group chat.
-    /^(.+?) invited (.+) to the group chat.   $/,
+    /^(.+?) invited (.+) to the group chat.\s+\s+\s+$/,
```

Comment:
Please make sure you understand what you are doing before you change anything... :(

Reply:
` {3}` or `\s{3}` should work if you want to match exactly three spaces :)